Basic authentication using devise

damwhit opened the pull request.

NickyBobby commented:
@erinnachen @damwhit @jedschneider Brian and I finished off the login authentication implementation using devise. We were a little thrown back by just how much devise does under the hood but it's pretty cool stuff. Jed do you mind taking a look at this PR and seeing if we set it up correctly?

damwhit commented:
basic auth complete with devise
removed devise views
removed devise controllers other than oauth
paired with jed to add pattern for page object model things
added sequencing for user model in factories.rb
still potentially need to comment things out in devise user migration

jedschneider commented:
👍  GTM
